doc/modules/mod_muc_commands.md

```diff
@@ -0,0 +1,47 @@
+# MongooseIM's multi-user chat commands set
+
+## Purpose
+This is a set of commands, providing actions connected with multi-user chat features.
+
+## Configuration
+This module contains command definitions which are loaded when the module is activated.
+There are no options to be provided, therefore the following entry in the config file is sufficient:
+
+```
+{mod_muc_commands, []}
+```
+
+## Commands
+This file consists of [commands definitions](http://mongooseim.readthedocs.io/en/latest/modules/mod_commands/).
+Following commands (along with functions necessary for them to run) are defined:
++ `create_muc_room`  
+Creates a MUC room.  
+    Args:  
+        - `host` (binary)  
+        - `name`  (binary) - room name  
+        - `owner` (binary) - the XMPP entity that would normally request an instant MUC room  
+        - `nick` (binary)  
++ `kick_user_from_room`  
+Kick a user from a MUC room (on behalf of a moderator).  
+    Args:  
+        - `host` (binary)  
+        - `name` (binary)  
+        - `nick` (binary)   
++ `invite_to_muc_room`  
+Sends a MUC room invite from one user to another.  
```

Comment:
Direct or mediated?
https://xmpp.org/extensions/xep-0045.html#invite
https://xmpp.org/extensions/xep-0249.html

Reply:
It is a direct invitation - I will make it clear in the doc